docker/qemu.sh

```diff
@@ -19,7 +19,7 @@ build_static_libffi () {
     tar --strip-components=1 -xzf "v${version}.tar.gz"
     ./configure --prefix="$td"/lib --disable-builddir --disable-shared --enable-static
     make "-j$(nproc)"
-    install -m 644 ./.libs/libffi.a /usr/lib64/
+    install -m 644 ./.libs/libffi.a /usr/local/lib/
```

Comment:
`/usr/lib64` is not a thing on `arm64`, so just use `/usr/local/lib` instead. This potentially could cause issues with folks who building of their old Pentium 4's, but I'm not sure. Either way, I'd reckon it's a small subset of the userbase anyway.